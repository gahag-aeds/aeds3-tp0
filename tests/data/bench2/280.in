280 140
63 76 35 21 95 8 18 88 27 60 61 38 73 78 72 17 99 36 58 62 0 82 24 2 79 43 12 55 57 83 20 73 59 7 94 6 15 12 95 94 25 8 85 50 38 57 67 37 93 77 51 46 11 75 48 90 70 12 45 28 47 17 1 59 76 47 65 91 60 12 38 37 20 23 87 10 80 7 99 26 36 2 72 48 78 72 38 0 36 36 80 84 5 33 95 82 81 60 25 93 73 63 30 45 86 17 8 19 76 7 45 13 10 69 61 40 41 51 40 77 87 73 13 45 6 8 27 87 21 52 80 46 68 62 43 54 32 51 25 8 59 70 73 21 39 34 61 32 38 53 10 77 26 75 22 85 36 1 72 57 6 5 55 74 67 98 80 99 50 6 60 61 76 33 82 68 20 95 0 58 0 62 35 79 90 10 64 26 63 88 35 69 93 90 95 61 88 76 12 38 82 72 51 10 6 85 30 78 32 83 36 33 45 23 64 35 85 28 61 49 16 96 18 62 86 14 75 27 42 87 17 76 60 69 38 18 6 69 96 39 52 84 24 49 59 88 37 45 68 98 94 36 47 64 98 85 30 73 64 24 61 82 0 73 3 39 43 9 60 91 0 64 75 24 13 34 64 50 79 84 
Sub 28 120
Sum 164 256
Sub 41 133
Min 183 275
Add 52 144
Max 174 266
Sub 144 236
Min 109 201
Sub 99 191
Min 1 93
Sub 15 107
Min 36 128
Add 157 249
Sum 125 217
Sub 98 190
Sum 48 140
Add 134 226
Min 25 117
Add 141 233
Min 2 94
Sub 85 177
Sum 52 144
Add 64 156
Sum 57 149
Sub 79 171
Max 81 173
Add 131 223
Min 131 223
Add 169 261
Max 186 278
Add 156 248
Sum 9 101
Sub 162 254
Max 51 143
Sub 46 138
Max 26 118
Sub 38 130
Min 3 95
Add 177 269
Sum 178 270
Sub 3 95
Sum 4 96
Add 26 118
Sum 14 106
Add 170 262
Sum 165 257
Add 61 153
Sum 159 251
Sub 2 94
Min 56 148
Sub 160 252
Min 86 178
Sub 149 241
Max 37 129
Add 184 276
Min 41 133
Add 159 251
Min 127 219
Sub 26 118
Max 182 274
Add 125 217
Sum 36 128
Add 4 96
Min 142 234
Sub 87 179
Sum 91 183
Sub 22 114
Max 124 216
Add 93 185
Min 53 145
Add 156 248
Min 137 229
Add 56 148
Min 181 273
Add 5 97
Min 81 173
Sub 158 250
Sum 107 199
Add 52 144
Max 159 251
Sub 163 255
Max 24 116
Add 99 191
Min 124 216
Sub 102 194
Sum 95 187
Sub 19 111
Max 115 207
Add 76 168
Sum 44 136
Add 110 202
Min 56 148
Add 122 214
Sum 155 247
Sub 10 102
Max 21 113
Sub 58 150
Sum 31 123
Sub 186 278
Max 151 243
Add 84 176
Max 154 246
Sub 142 234
Min 181 273
Sub 2 94
Min 147 239
Sub 116 208
Max 159 251
Add 66 158
Min 167 259
Add 172 264
Min 83 175
Sub 46 138
Sum 83 175
Add 62 154
Max 2 94
Add 32 124
Min 80 172
Add 117 209
Min 149 241
Add 166 258
Max 81 173
Sub 17 109
Sum 142 234
Add 48 140
Max 68 160
Sub 162 254
Sum 147 239
Sub 141 233
Max 15 107
Sub 152 244
Min 79 171
Sub 23 115
Max 107 199
Sub 114 206
Sum 24 116
Sub 108 200
Min 43 135
Add 37 129
Sum 28 120
