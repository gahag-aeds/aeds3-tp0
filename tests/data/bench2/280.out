5228
0
98
-1
-1
-2
4659
4185
-2
-2
4112
4132
97
0
100
4456
97
96
-5
5313
4174
4282
5566
5457
-5
-5
95
-5
0
102
3931
1
4302
102
-7
2
1
-5
4701
102
97
2
4545
101
4244
-4
5367
96
4038
102
102
0
4
100
3
-5
4562
96
-4
5
101
5381
96
5273
96
-4
100
3934
-5
4110
